app/src/main/java/com/cornellappdev/scoop/ui/components/post/TimePicker.kt

```diff
@@ -1,6 +1,5 @@
 package com.cornellappdev.scoop.ui.components.post
 
-import android.R
```

Comment:
this R file is usually com.(project name here).R

Reply:
I removed this import!